app-misc/lcd4linux/lcd4linux-0.11.0_pre20170527.ebuild

```diff
@@ -0,0 +1,201 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit autotools flag-o-matic python-single-r1
+
+DESCRIPTION="A small program that grabs information and displays it on an external LCD"
+HOMEPAGE="https://lcd4linux.bulix.org/"
+SRC_URI="https://www.bl4ckb0x.de/files/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
```

Comment:
order keywords alphabetically please.

Reply:
Whoops, thanks for the catch. Fixed by newest push.